2025/01/26 02:05:03 [error] 396#396: *854 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "OPTIONS /connect/token HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/token", host: "api.getabp.net", referrer: "https://admin.getabp.net/"
2025/01/26 02:05:03 [error] 396#396: *857 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "GET /connect/authorize?response_type=code&client_id=Abp_App&state=dDhCaERCbFQ0QXlZUzV5TUNaMkFEMW1OMlBvdE14YldWYXZwdX5oUEdMX21q&redirect_uri=https%3A%2F%2Fadmin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=XMNx0vcfedDJk3QWyBia8Nxtv9nJiG5HfDbmOz4KxtE&code_challenge_method=S256&nonce=dDhCaERCbFQ0QXlZUzV5TUNaMkFEMW1OMlBvdE14YldWYXZwdX5oUEdMX21q&culture=zh-Hant&ui-culture=zh-Hant HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/authorize?response_type=code&client_id=Abp_App&state=dDhCaERCbFQ0QXlZUzV5TUNaMkFEMW1OMlBvdE14YldWYXZwdX5oUEdMX21q&redirect_uri=https%3A%2F%2Fadmin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=XMNx0vcfedDJk3QWyBia8Nxtv9nJiG5HfDbmOz4KxtE&code_challenge_method=S256&nonce=dDhCaERCbFQ0QXlZUzV5TUNaMkFEMW1OMlBvdE14YldWYXZwdX5oUEdMX21q&culture=zh-Hant&ui-culture=zh-Hant", host: "api.getabp.net", referrer: "https://admin.getabp.net/"
2025/01/26 04:37:01 [error] 396#396: *3892 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "OPTIONS /connect/token HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/token", host: "api.getabp.net", referrer: "https://admin.getabp.net/"
2025/01/26 04:37:01 [error] 396#396: *3892 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "OPTIONS /api/abp/application-configuration?includeLocalizationResources=false HTTP/1.1", upstream: "https://192.168.65.2:44366/api/abp/application-configuration?includeLocalizationResources=false", host: "api.getabp.net", referrer: "https://admin.getabp.net/"
2025/01/26 04:37:01 [error] 396#396: *3896 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "GET /connect/authorize?response_type=code&client_id=Abp_App&state=MXZzcDVNUkJiYmNKQUdBMGx2NkstVTV0OG96T1dyQlU4dnJmZzFBZ1Z6LTlP&redirect_uri=https%3A%2F%2Fadmin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=yTRKZTHJAPu96981-8QNnEr459GYHe9TR6vvGepJjmU&code_challenge_method=S256&nonce=MXZzcDVNUkJiYmNKQUdBMGx2NkstVTV0OG96T1dyQlU4dnJmZzFBZ1Z6LTlP&culture=zh-Hant&ui-culture=zh-Hant HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/authorize?response_type=code&client_id=Abp_App&state=MXZzcDVNUkJiYmNKQUdBMGx2NkstVTV0OG96T1dyQlU4dnJmZzFBZ1Z6LTlP&redirect_uri=https%3A%2F%2Fadmin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=yTRKZTHJAPu96981-8QNnEr459GYHe9TR6vvGepJjmU&code_challenge_method=S256&nonce=MXZzcDVNUkJiYmNKQUdBMGx2NkstVTV0OG96T1dyQlU4dnJmZzFBZ1Z6LTlP&culture=zh-Hant&ui-culture=zh-Hant", host: "api.getabp.net", referrer: "https://admin.getabp.net/"
2025/01/26 04:44:55 [error] 396#396: *3939 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "OPTIONS /connect/token HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/token", host: "test1.api.getabp.net", referrer: "https://test1.admin.getabp.net/"
2025/01/26 04:44:55 [error] 396#396: *3939 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "OPTIONS /api/abp/application-configuration?includeLocalizationResources=false HTTP/1.1", upstream: "https://192.168.65.2:44366/api/abp/application-configuration?includeLocalizationResources=false", host: "test1.api.getabp.net", referrer: "https://test1.admin.getabp.net/"
2025/01/26 04:44:55 [error] 396#396: *3942 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "GET /connect/authorize?response_type=code&client_id=Abp_App&state=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&redirect_uri=https%3A%2F%2Ftest1.admin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=0avcwD2BBm2r4iWyAGPrl4wXrtN8lb2UAeN7F5fsbeA&code_challenge_method=S256&nonce=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&culture=zh-Hant&ui-culture=zh-Hant HTTP/1.1", upstream: "https://192.168.65.2:44366/connect/authorize?response_type=code&client_id=Abp_App&state=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&redirect_uri=https%3A%2F%2Ftest1.admin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=0avcwD2BBm2r4iWyAGPrl4wXrtN8lb2UAeN7F5fsbeA&code_challenge_method=S256&nonce=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&culture=zh-Hant&ui-culture=zh-Hant", host: "test1.api.getabp.net", referrer: "https://test1.admin.getabp.net/"
2025/01/26 04:44:55 [error] 396#396: *3942 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: *.api.getabp.net, request: "GET /favicon.ico HTTP/1.1", upstream: "https://192.168.65.2:44366/favicon.ico", host: "test1.api.getabp.net", referrer: "https://test1.api.getabp.net/connect/authorize?response_type=code&client_id=Abp_App&state=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&redirect_uri=https%3A%2F%2Ftest1.admin.getabp.net&scope=openid%20offline_access%20Abp&code_challenge=0avcwD2BBm2r4iWyAGPrl4wXrtN8lb2UAeN7F5fsbeA&code_challenge_method=S256&nonce=LkJ1MlJEV1dGWnRYS0pxazBNUXYuOWc2ajRoMGI5YU80TTR1X3pucnRDU2d1&culture=zh-Hant&ui-culture=zh-Hant"
